README.md

```diff
@@ -22,7 +22,8 @@ References for developing libraries for Processing can be found on the following
 - https://github.com/benfry/processing4/wiki/Library-Overview
 
 > [!Note]
-> This template is based on Gradle. If you are looking for the old Ant-based template, see [processing/processing-library-template-ant](processing/processing-library-template-ant)
+> This template is based on Gradle. If you are looking for the old Ant-based template, see [https://github.com/processing/processing-library-template-ant](processing/processing-library-template-ant)
```

Comment:
HI @alexdmiller and thanks for catching the broken link! On closer inspection, it looks like the `[]` text was fine as it was! The issue is actually with the `()` part, which still points to the wrong URL. The part in parenthesis is what needs to be updated to `https://github.com/processing/processing-library-template-ant` to fix the link. Would you mind amending your PR?